ra/SAPHana

```diff
@@ -2873,8 +2868,7 @@ SAPSTARTSRV=""
 SAPCONTROL=""
 DIR_PROFILE=""
 SAPSTARTPROFILE=""
-SAPHanaFilter="ra-act-dec-lpa"
-
+SAPHanaFilter="RA-ACT-DEC-LPA"
```

Comment:
This change has some problems.
1. The filter is a least referenced in some manual pages, which need to adapt too. And I'm pretty sure there will be a lot of training material, conference slides and other documentation referencing this filter value.
2. This change may result in trouble with some customers as the filter value may be stored as an attribute inside the cluster. And if such an attribute exists, the RA is using the value from the cluster. Which means that the new filter handling will fail as it relies on upper case filter value. So the change is not compatible.